src/pytest_split/plugin.py

```diff
@@ -181,6 +185,60 @@ def pytest_collection_modifyitems(
         return None
 
 
+def _reorganize_broken_up_ipynbs(group: "TestGroup", items: list) -> None:
```

Comment:
More accurate type for `items` is probably `List[nodes.Item]`?

Reply:
Thanks for the thorough review :) I just changed it and the linter for some reason is still not happy. Here's the error I'm getting:
```python
tests/test_ipynb.py:57: error: Argument 2 to "_reorganize_broken_up_ipynbs" has incompatible type "List[item]"; expected "List[Item]"  [arg-type]
tests/test_ipynb.py:68: error: Argument 2 to "_reorganize_broken_up_ipynbs" has incompatible type "List[item]"; expected "List[Item]"  [arg-type]
tests/test_ipynb.py:77: error: Argument 2 to "_reorganize_broken_up_ipynbs" has incompatible type "List[item]"; expected "List[Item]"  [arg-type]
Found 3 errors in 1 file (checked 9 source files)
```

Reply:
That's probably because the test seems to be using `item` type which is defined by you: https://github.com/jerry-git/pytest-split/blob/2c240f840f312c472e63e45249ee7aab9eeb795c/tests/test_ipynb.py#L8

Reply:
Right, but I copied that from `test_algorithms.py`, and the linter seems to be happy with that. I'm not very familiar with mypy :) is there a way to fix it?

Reply:
You can leave the type as `list` for now 🙂

Reply:
after this one should be ready to go, i.e. just revert back to `items: list` to get the CI green

Reply:
Done, should be ready to go!